docs/user-guide/cli-using-using-profiles.md

```diff
@@ -20,20 +20,20 @@ Zowe CLI contains the following types of profiles:
 - Profiles are **not** required. You can choose to specify all connection details for every command.
 - Profile values are stored on your computer in plaintext in `C:\Users\<yourUsername>\.zowe\profiles` (Windows) or in `~/.zowe/profiles` (Mac/Linux).
 
-### Important information about V2 profiles
+### Important information about team profiles
 
 - With the introduction of [team profiles](../user-guide/cli-using-using-team-profiles.md), the Secure Credential Store (SCS) Plug-in is deprecated. Secure credential encryption is now handled by the the secure array in the `zowe.config.json` file.
-- You can convert all of your Zowe CLI and Zowe CLI plug-ins V1 profiles to V2 profiles by issuing the following command:
+- You can convert all of your Zowe CLI and Zowe CLI plug-ins V1 profiles to team profiles by issuing the following command:
 
     ```
     zowe config convert-profiles
     ```
 
-    **Note:** You can continue using Zowe CLI and Zowe CLI plug-ins V1 profiles with Zowe CLI V2. However, we highly recommend that you implement V2 profiles with Zowe CLI V2.
+    **Note:** You can continue using Zowe CLI and Zowe CLI plug-ins V1 profiles with Zowe CLI team profiles. However, we highly recommend that you implement team profiles with Zowe CLI V2.
```

Comment:
I would leave the first instance as `You can continue using Zowe CLI and Zowe CLI plug-ins V1 profiles with Zowe CLI V2`

Reply:
updated ....